django/contrib/admin/templates/admin/base.html

```diff
@@ -10,6 +10,9 @@
 {% block extrastyle %}{% endblock %}
 {% if LANGUAGE_BIDI %}<link rel="stylesheet" type="text/css" href="{% block stylesheet_rtl %}{% static "admin/css/rtl.css" %}{% endblock %}">{% endif %}
 {% block extrahead %}{% endblock %}
+{% if not is_popup and is_nav_sidebar_enabled %}
+<script src="{% static 'admin/js/nav_sidebar.js' %}" async></script>
```

Comment:
The script contains the code:

```js
const toggleNavSidebar = document.getElementById('toggle-nav-sidebar');
```

Now that script is loaded in `<head>` the above code shouldn't trigger until after the DOM is ready. If the JS loads before the DOM, the `getElementById()` could return `null` (I experienced this during testing). This wasn't required with the old version as the JS was loaded after the DOM queried element. Using `defer` might be an alternative approach to handling it.